### ISSUE TITLE
`ts-force-gen` property renaming to relationName is not working

### ISSUE THREAD

**Issue by jesse23** [ChuckJonas/ts-force#106]
## Use case
- Given `Account` type with relationship `Account.Owner -> User`. With API Definition below:

```json
{
  "sObjects": [
    {
      "apiName": "Account",
      "className": "Account",
      "fieldMappings": [
        {
          "apiName": "Owner",
          "propName": "myOwner"
        }
     ]
  }]
}
```

## Result
It fails to pick the property definition.

## Root cause
For code below, we should use `sobConfig` rather than `relatedSobConfig`, right? 
https://github.com/ChuckJonas/ts-force/blob/3098d855495c2dca27ff8c9c7345a1346cd37c05/ts-force-gen/src/sObjectGenerator.ts#L401

## Workaround ( which is just for inference )
Based on the analysis, if we put the fildMapping to user, it works....

- Config
```json
      "className": "User",
      "apiName": "User",
      "fieldMappings": [
        {
          "apiName": "Owner",
          "propName": "myOwner"
        }
      ]
```

- Outcome
```typescript
    @sField({ apiName: 'Owner', createable: false, updateable: false, required: false, reference: () => { return User }, childRelationship: false, salesforceType: SalesforceFieldType.REFERENCE, salesforceLabel: 'Owner ID', externalId: false })
    public myOwner?: User;
```

**Comment by ChuckJonas** [comment]
I think you're right.  It looks like it got changed during a recent refactor... Not sure why.

**Comment by jesse23** [comment]
Will put a merge request tomorrow if u don't mind. Thx!

I've noticed some of the forks have the correct impl, not sure that is before refactor or they fix at their own branch :)

**Comment by ChuckJonas** [comment]
Sure, I should have time to release a new NPM package tomorrow.  There are a couple other minor updates that I need to release anyways.

Thanks!

(FYI: I'm pretty all the forks are all very stale).

**Comment by jesse23** [comment]
https://github.com/ChuckJonas/ts-force/pull/107

Thx!

**Comment by ChuckJonas** [comment]
released in v3.0.2
